phpunit.xml.dist

```diff
@@ -9,7 +9,7 @@
          processIsolation="false"
          stopOnFailure="false"
          syntaxCheck="false"
-         bootstrap="Tests/bootstrap.php"
+         bootstrap="tests/bootstrap.php"
```

Comment:
yep. looks like the sphinx build has a hardcoded path. we could keep Resources/doc outside of src/ - while this seems not common practice, it would actually make sense to me, i see no point in putting that inside the source code part.